src/test/java/org/assertj/core/error/ShouldHaveSameFileSystemAs_create_Test.java

```diff
@@ -12,15 +12,16 @@
  */
 package org.assertj.core.error;
 
-import static java.lang.String.format;
-import static org.assertj.core.api.BDDAssertions.then;
-import static org.assertj.core.error.ShouldHaveSameFileSystemAs.shouldHaveSameFileSystemAs;
+import org.assertj.core.description.TextDescription;
+import org.junit.jupiter.api.Test;
 
+import java.io.File;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import org.assertj.core.description.TextDescription;
-import org.junit.jupiter.api.Test;
+import static java.lang.String.format;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldHaveSameFileSystemAs.shouldHaveSameFileSystemAs;
```

Comment:
revert all these changes, this has nothing to do with your PR

Reply:
These changes are to fix the bugs in these tests(Some checks are failed due to this bug). And this just change the import sequence. It should not have any other impact?😂

Reply:
this PR is about float/double not `ShouldHaveSameFileSystemAs`, don't mix things up.

and TBH it's not funny for me to spend so much time reviewing this PR.